lakefile.lean

```diff
@@ -48,3 +48,53 @@ lean_lib Archive where
 
 lean_lib Counterexamples where
   roots := #[`Counterexamples]
+
+section Scripts
+
+open System
+
+def System.FilePath.makeRelativeTo (dir file : FilePath) : Option FilePath := do
```

Comment:
Should this function be moved elsewhere?

Reply:
Personally, I would suggest using `readDir` instead of `walkDir` so you can avoid this post-processing.